dnf/logging.py

```diff
@@ -151,6 +152,11 @@ def setup(self, verbose_level, error_level, logdir):
         logger_warnings.addHandler(self.stderr_handler)
         logger_warnings.addHandler(handler)
 
+        # setup LIBREPO logging
+        if verbose_level >= DDEBUG:
```

Comment:
we should log this in any case. Remove the condition, please.

WRT [bug assigned to you](https://bugzilla.redhat.com/show_bug.cgi?id=1196629) it should have printed debug logs from librepo to stdout in verbose mode. But talking to Tomas it's not a good idea to print it all - only the important ones (connected/failure callbacks) and what they request in the report. Do this in another commit, please. You have done so far the most of them IIRC.